### ISSUE TITLE
Initial audio preference popup prevents access to accessibility button.

### ISSUE THREAD

**Issue by dancgray** [deltanet/adapt-audio#161]
Really users should be able to tab to the accessibility button first before tabbing to the audio continue or disable audio button.

Currently they tab first to the audio popup.

This may also be the same for bookmarking initial preference popup.

**Comment by dancgray** [comment]
No longer relevant in FW v4.
